colearn_examples/ml_interface/mli_fraud.py

```diff
@@ -105,6 +105,17 @@ def mli_accept_weights(self, weights: Weights):
         self.set_weights(weights)
         self.vote_score = self.test(self.vote_data, self.vote_labels)
 
+    def mli_get_current_model(self) -> ColearnModel:
+        """
+        :return: The current model and its format
+        """
+
+        return ColearnModel(
+            model_format=ModelFormat(2),
```

Comment:
ModelFormat.ONNX would be clearer here, but it's minor.

Reply:
Fair, done